Tests/NetworkServiceTests/NetworkErrorTests/NetworkError+LocalizedErrorTests.swift

```diff
@@ -0,0 +1,40 @@
+import NetworkService
+import XCTest
+
+final class NetworkErrorLocalizedErrorTests: XCTestCase {
+    func testLocalizedDescriptionReturnsCorrespondingMessageForBadRequestError() {
+        let expectedLocalizedDescription = "The server cannot process the request due to an error in the request."
+        let actualLocalizedDescription = NetworkError.badRequest.localizedDescription
+        XCTAssertEqual(expectedLocalizedDescription, actualLocalizedDescription)
+    }
```

Comment:
Can we do simpler?
```suggestion
    func testLocalizedDescriptionReturnsCorrespondingMessage() {
        XCTAssertEqual(
            NetworkError.badRequest.localizedDescription,
            "The server cannot process the request due to an error in the request."
        )
        
        XCTAssertEqual(NetworkError.clientError.localizedDescription,
                       "The server cannot process the request due to a client error."
        )
    }
```

Reply:
I don't think it's much easier. For me the variant with constants looks more clear.